modules/ProfileInfo.js

```diff
@@ -0,0 +1,52 @@
+
+
+const ProfileInfo = () => 
+       {
```

Comment:
Let's move this opening curly brace to `line 3`

This: `const ProfileInfo = () => {`